outputs/policyadapter.go

```diff
@@ -0,0 +1,124 @@
+package outputs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/anushkamittal20/falcoadapter/pkg/apis/wgpolicyk8s.io/v1alpha2"
+	clusterpolicyreport "github.com/anushkamittal20/falcoadapter/pkg/apis/wgpolicyk8s.io/v1alpha2"
```

Comment:
why your repo is the source? can't we follow k8s upstream?

Reply:
Hey Thomas, this should be temporary as k8s doesnt have the generated CRD client codes yet but it is in proess to bring those generated CRD client codes to k8s official repo (https://github.com/kubernetes-sigs/wg-policy-prototypes/issues/79) and then we could make those changes.

Reply:
ok, I see. We'll discuss about on Slack I think